etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/orders/OrderControllerTest.groovy

```diff
@@ -16,6 +17,7 @@ class OrderControllerTest extends Specification {
         TestApplicationContext.init()
         TestApplicationContext.register(OrderController, OrderController.getInstance())
         TestApplicationContext.register(MetricMetadata, Mock(MetricMetadata))
+        TestApplicationContext.register(RuleEngine, Mock(RuleEngine))
```

Comment:
We should probably put an assertion in some test (or a new test) that ensures we call the `RuleEngine`.  If we accidentally remove the call to the rules engine, I would hope a unit test would fail.

Reply:
Added an assertion to the mock on the happy path test